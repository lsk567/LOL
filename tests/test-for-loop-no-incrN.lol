for (int i = 0; i < 10;  ) {
    println("hello " + str_of_int(i));
    i = i + 2;
}
