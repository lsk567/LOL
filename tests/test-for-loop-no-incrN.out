hello 0
hello 2
hello 4
hello 6
hello 8
